client/chainlink_models.go

```diff
@@ -654,3 +654,17 @@ func ObservationSourceSpecBridge(bta BridgeTypeAttributes) string {
 		parse [type=jsonparse path="data,result"];
 		fetch -> parse;`, bta.Name, bta.RequestData)
 }
+
+// marshallTemplate Helper to marshall templates
+func marshallTemplate(jobSpec interface{}, name, templateString string) (string, error) {
```

Comment:
Actually, now it's two of them

Reply:
Yeah, I didn't want to create a separate package or just a utils for a simple go templates func that'll probably never change.